T-Dict/T-Dict/Controller/DetailsVC/DetailsViewController.swift

```diff
@@ -45,6 +45,10 @@ class DetailsViewController: UIViewController {
         antonymsVC.view.tag = 2
         rhymeVC.view.tag = 3
         childViews = [defineVC, synonymsVC, antonymsVC, rhymeVC]
+        defineVC.config(word: word ?? "")
+        synonymsVC.config(word: word ?? "")
+        antonymsVC.config(word: word ?? "")
+        rhymeVC.config(word: word ?? "")
```

Comment:
```suggestion
        childViews.forEach {
            $0.config(....)
        }
```

Reply:
về sau mỗi màn hình này sẽ đc config bằng những data khác nhau nên e k dùng for each ạ

Reply:
@minhtien1403 
nếu vậy thì tách riêng ra 1 func configPages nhé